packages/react-native/README.md

```diff
@@ -0,0 +1,158 @@
+# LavaMoat React Native lockdown
+
+LavaMoat React Native lockdown sets up [Hardened JavaScript][hardened-js-ext] in React Native apps.
+
+This is done by repairing Hermes/JSC intrinsics at Metro's serializer option _getPolyfills_:
+
+> An optional list of polyfills to include in the bundle. The list defaults to a set of common polyfills for Number, String, Array, Object...
+
+Then hardening the intrinsics at Metro's first serializer option _getRunModuleStatement_:
+
+> Specify the format of the initial require statements that are appended at the end of the bundle. By default is `\_\_r(${moduleId});`.
+
+which runs `lockdown` JS before React Native [sets up][init-core-ext] global variables typical in most JavaScript environments.
+
+## Install
+
+```shell
+npm i @lavamoat/react-native
+```
+
+or
+
+```shell
+yarn add @lavamoat/react-native
+```
+
+## Usage
+
+### Babel config
+
+```js
+// babel.config.js
+module.exports = {
+  ignore: [/ses\.cjs/, /ses-hermes\.cjs/],
+  presets: ['module:metro-react-native-babel-preset'],
+}
+```
+
+### Metro config
+
+#### Hermes
+
+```js
+// metro.config.js
+const { getDefaultConfig, mergeConfig } = require('@react-native/metro-config')
+const { lockdownSerializer } = require('@lavamoat/react-native')
+
+/**
+ * Metro configuration https://reactnative.dev/docs/metro
+ *
+ * @type {import('@react-native/metro-config').MetroConfig}
+ */
+const config = {
+  serializer: lockdownSerializer(
+    {
+      hermesRuntime: true,
+    },
+    {
+      // your previous serializer config if any, for example:
+      getPolyfills: () => {
+        return [
+          ...require('@react-native/js-polyfills')(),
+          require.resolve('reflect-metadata'),
+        ]
+      },
+    }
+  ),
+}
+
+module.exports = mergeConfig(getDefaultConfig(__dirname), config)
+```
+
+#### JSC
```

Comment:
https://reactnative.dev/blog/2025/04/08/react-native-0.79#jsc-moving-to-community-package